features/src/icc/devcontainer-feature.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Intel dpcpp toolchain",
+  "id": "icc",
+  "version": "23.6.0",
+  "description": "A feature to install the Intel dpcpp toolchain",
+  "options": {
+    "version": {
+      "type": "string",
+      "proposals": [
+        "2022.2.0",
+        "2022.2.1",
+        "2023.0.0",
+        "2023.1.0"
+      ],
+      "default": "2022.2.1",
```

Comment:
Any reason the default is 2022 and not 2023?

Reply:
CTK doesn't yet support 2023 if I recall correctly.